pkg/controller/backup_controller.go

```diff
@@ -654,9 +653,14 @@ func (b *backupReconciler) runBackup(backup *pkgbackup.Request) error {
 		selector := label.NewSelectorForBackup(backup.Name)
 		vscList := &snapshotv1api.VolumeSnapshotContentList{}
 
-		volumeSnapshots, err = b.waitVolumeSnapshotReadyToUse(context.Background(), backup.Spec.CSISnapshotTimeout.Duration, backup.Name)
-		if err != nil {
-			backupLog.Errorf("fail to wait VolumeSnapshot change to Ready: %s", err.Error())
+		if b.volumeSnapshotLister != nil {
```

Comment:
Can we move the `ResetVolumeSnapshotContent` to plugin as well since it is also part of CSI backup workflow?
In this way, the entire `if features.IsEnabled(velerov1api.CSIFeatureFlag)` block will be removed from Velero code

Reply:
`ResetVolumeSnapshotContent` can be removed, but the rest of code block is still needed. The VolumeSnapshot, VolumeSnapshotContent, VolumeSnapshotClass lists are needed to persist backup data.

Reply:
OK, then after we remove the individual VolumeSnapshot, VolumeSnapshotContent, VolumeSnapshotClass items in backup store, we will be able to remove the code block here.